wazimap/data/tables.py

```diff
@@ -605,9 +607,8 @@ def _build_model_from_fields(self, fields, table_name, geo_level=None, value_typ
 
         # create the table model
         class Model(Base):
-            table = table_name
+            table = db_table
             __table__ = Table(table, Base.metadata, *table_args, extend_existing=True)
```

Comment:
We could probably just do:
`Table(db_table, Base.metadata, *table_args, extend_existing=True)`